packages/components/src/internal/components/user/UserProfile.tsx

```diff
@@ -62,10 +65,11 @@ export class UserProfile extends PureComponent<Props, State> {
             removeCurrentAvatar: false,
             reloadRequired: false,
             hasError: false,
+            groups: undefined,
         };
     }
 
-    componentDidMount = (): void => {
+    componentDidMount = async (): Promise<void> => {
         getQueryDetails(SCHEMAS.CORE_TABLES.USERS)
```

Comment:
It's a bit weird to mix and match promises and async code. We should really be doing all of one type, preferably async, so I think we should update the call to getQueryDetails here to use `await`.